templates/openshift/common/utils/get_deployment_stamp_from_route.yml.j2

```diff
@@ -1,7 +1,9 @@
 apiVersion: v1
 kind: Route
 metadata:
-  name: "{{ app.name }}-{{ prefix }}"
+  # We only keep track of the first hostname found as we want to fetch the
+  # deployment_stamp that is common to every routes for a deployment
```

Comment:
I don't understand this comment.
What are the other hostnames that I could get?

Reply:
An application can define several routes (_e.g._ two services with two different routes). Our goal here is to get the `deployment_stamp` of a deployed stack by getting a route object. As the `deployment_stamp` is commonly shared among objects of a stack, there is no need to get more than one route to achieve this. Is that more clear?

Reply:
Ok! The key to understanding is "Our goal here is to get the deployment_stamp of a deployed stack by getting a route object". Can you insert your whole answer as a comment?